biosys/urls.py

```diff
@@ -37,12 +38,12 @@ def admin_view_selection_view(request):
 
 web_urls = [
     # Authentication URLs
-    url(r'^logout/$', auth_views.logout, {'next_page': '/login/'}, name='logout'),
+    url(r'^logout/$', LogoutView.as_view(), {'next_page': '/login/'}, name='logout'),
```

Comment:
ideally `url()` should be replaced with `re_path()`.  It's currently an alias for `re_path()` in Django 3.x but it will be removed in Django 4.  
https://github.com/django/django/blob/master/django/conf/urls/__init__.py

Reply:
Ok, I'll make the change.